docs/robots/jackal/tutorials_jackal.mdx

```diff
@@ -4,9 +4,1317 @@ sidebar_position: 3
+Windows on your Jackal, installing ROS & Jackal ROS components, updating firmware and configuring Windows
+for use.
+
+The following instructions assume that the Jackal is connected to a keyboard and mouse during setup. Once
+configured, these can be removed and remote access can be leveraged.
+
+For more information on ROS on Windows, please visit [Microsoft's ROS documentation](http://aka.ms/ros).
+
+#### Installing Windows 10
+
+ROS1 is supported by Microsoft for Windows 10 Desktop and Windows 10 IoT Enterprise. Windows 10 IoT Enterprise
+is an embedded version of Windows 10 for dedicated purpose devices, such as a Robot.
+
+1. You can [download a trial](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-10-enterprise) of Windows 10 IoT Enterprise Long Term Service (LTSC) x64 build, use a Windows license from your Enterprise agreement, or a commercially purchased version of Windows 10 Desktop.
+2. Create a [USB Boot stick](https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/install-windows-from-a-usb-flash-drive) and copy the contents of the ISO generated from the image above.
+3. Boot Windows and install following the prompts.
+4. Install ROS on Windows using the [ROS Wiki instructions](https://wiki.ros.org/Installation/Windows).
+5. Please follow the instructions closely, as ROS is sensitive to the operating environment.
+
+This will install the development platform on the Jackal, which allows you to develop your robotics solutions. Once you've
+finalized your development, you can build a deployment package and deploy your solutions using Chocolatey.
+
+If you have Windows specific questions or problems, Microsoft offers [community support through Github](https://github.com/ms-iot/rosonwindows) and [paid support](http://aka.ms/ros/support).
+
+#### Installing Jackal Software
+
+Once you've installed Windows and ROS, It's time to install the Jackal software. Please open an Administrative ROS command prompt created during the ROS install.
+
+```
+:: Only need to install wget once.
+choco install wget
+:: Close and Reopen the command Window.
+
+mkdir c:\ws
+cd c:\ws
+wget https://raw.githubusercontent.com/jackal/jackal-sphinx-tutorials/master/jackal.rosinstall
+wstool init src jackal.rosinstall
+
+:: Later versions of pyproj are not deployed as a binary and do not build automatically
+pip install pyproj==1.9.6
+
+rosdep update
+rosdep install --from-paths src --ignore-src -r -y
+catkin_make_isolated
+devel_isolated\setup.bat
+```
+
+:::note
+
+rosdep will not find several components. These include rosserial_server, robot_upstart, joint_state_publisher_gui and others. gmapping is coming soon.
 
 :::
+
+:::note
+
+The Jackal ROS Nodes and support software is distributed as source code on Github. Other Clearpath repositories have not been enabled as of April 2020, but will be evaluated based on demand.
+
+:::
+
+#### MCU Firmware Update
```

Comment:
Line 574 of this file has similar MCU Firmware Update instructions. Can these be condensed into a single section?

Reply:
If not; consider adding some reference to Windows 10 for all the subheadings to reduce confusion for people skimming the page: 
'Installing Jackal Software, on Windows 10'
'MCU Firmware Update, on Windows 10'

Reply:
As discussed, I removed the Windows 10 section entirely.